sdcm/kafka/kafaka_cluster.py

```diff
@@ -0,0 +1,158 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+
+import logging
+from pathlib import Path
+from functools import cached_property
+
+import requests
+
+from sdcm import cluster
+from sdcm.wait import wait_for
+from sdcm.remote import LOCALRUNNER
+from sdcm.utils.git import clone_repo
+from sdcm.utils.common import get_sct_root_path
+from sdcm.utils.remote_logger import CommandClusterLoggerBase
+
+# TODO: write/think more about the consumers
+
+logger = logging.getLogger(__name__)
+
+
+class KafkaLogger(CommandClusterLoggerBase):
```

Comment:
Other such loggers are placed in the `sdcm.utils.remote_logger`.
So, why this one is here?

Reply:
no particular reason, it was easier to develop it like that.
Also it's a bit different that the other ones, since it's not being initiate in the same fashion.

I can move, probably when we'll have more kind of this setup, like on VM, or as SaaS this would have more variants

Reply:
move to remote_logger, even that it's not exactly remote :)